### ISSUE TITLE
[BUG] Wiki documentation error

### ISSUE THREAD

**Issue by adripo** [Tzahi12345/YoutubeDL-Material#679]
**Describe the bug**
The environment variable [ytdl_write_config](https://github.com/Tzahi12345/YoutubeDL-Material/wiki/Environment-Variables#ytdl_write_config) specified in the wiki should be [write_ytdl_config](https://github.com/Tzahi12345/YoutubeDL-Material/blob/master/docker-compose.yml#L8) as specified in the docker-compose file.

**Comment by Tzahi12345** [comment]
Updated wiki after #680 merged! Removed that section as it's no longer needed. Thanks for pointing this out regardless :)
